x-pack/plugins/cases/public/components/custom_fields/text/edit.tsx

```diff
@@ -46,27 +58,38 @@ const FormWrapperComponent: React.FC<FormWrapper> = ({
   isLoading,
   onChange,
 }) => {
-  const { form } = useForm({
-    defaultValue: { value: initialValue },
+  const { form } = useForm<{ value: string }>({
+    defaultValue: {
+      value:
+        typeof customFieldConfiguration?.defaultValue === 'string' && isEmpty(initialValue)
```

Comment:
I think that `typeof customFieldConfiguration?.defaultValue === 'string'` is redundant. Wdyt about casting?

Reply:
What about `customFieldConfiguration?.defaultValue != null && isEmpty(initialValue)`?